### PR TITLE
Added derivative of tanh

### LINKED ISSUE #60: Convert Activations from simple functions to classes and add gradient method.
Currently, all the activations in activations.py are simple functions. 

However, for future implementation of Neural Networks, we will also need derivatives methods of each of these
functions. You can have a look into loss_func.py for reference. There, each class represents a loss function and has both loss
and derivative methods. You have to implement something similar for activations.

### PR DESCRIPTION
**Pull request by Abjcodes**
#### References to other Issues or PRs or Relevant literature

Fixes #60 

#### Brief description of what is fixed or changed
Created TanH class with activation and derivative methods. The derivative method returns the array of derivatives of hyperbolic tangent function on X Vector.

#### Other comments

### CODE REVIEW

**Comment by rohansingh9001** [comment]
@Abjcodes while the code and formatting and docstrings are correct, tanh refers to the hyperbolic tangent function and not the inverse of a normal trigonometric tangent. Please make the changes in your docstrings. Rest LGTM

**Comment by Abjcodes** [comment]
> @Abjcodes while the code and formatting and docstrings are correct, tanh refers to the hyperbolic tangent function and not the inverse of a normal trigonometric tangent. Please make the changes in your docstrings. Rest LGTM

Sure. I have made the changes.

**Review by rohansingh9001 — APPROVED** [review]
Approved and merged
